README.md

```diff
@@ -1,27 +1,29 @@
-# Virtual Schemas
+# Common module of Exasol JDBC Adapter for Virtual Schemas
 
-[![Build Status](https://travis-ci.org/exasol/virtual-schemas.svg?branch=master)](https://travis-ci.org/exasol/virtual-schemas)
+This is one of the modules of Exasol JDBC Adapter for Virtual Schemas.
+It's not supposed to be used by itself, separately.
+You can find the full description of the project here: https://github.com/exasol/virtual-schemas
 
-<p style="border: 1px solid black;padding: 10px; background-color: #FFFFCC;"><span style="font-size:200%">&#9888;</span> Please note that this is an open source project which is officially supported by Exasol. For any question, you can contact our support team.</p>
 
-Virtual schemas provide a powerful abstraction to conveniently access arbitrary data sources. Virtual schemas are a kind of read-only link to an external source and contain virtual tables which look like regular tables except that the actual data are not stored locally.
+## Dependencies
 
-After creating a virtual schema, its included tables can be used in SQL queries and even combined with persistent tables stored directly in Exasol, or with other virtual tables from other virtual schemas. The SQL optimizer internally translates the virtual objects into connections to the underlying systems and implicitly transfers the necessary data. SQL conditions are tried to be pushed down to the data sources to ensure minimal data transfer and optimal performance.
+### Run Time Dependencies
 
-That's why this concept creates a kind of logical view on top of several data sources which could be databases or other data services. By that, you can either implement a harmonized access layer for your reporting tools or you can use this technology for agile and flexible ETL processing, since you don't need to change anything in Exasol if you change or extend the objects in the underlying system.
+| Dependency                                                                   | Purpose                                                | License                       |
+|------------------------------------------------------------------------------|--------------------------------------------------------|-------------------------------|
+| [GlassFish](https://javaee.github.io/jsonp/)                                 | JSON Processing                                        | CDDL-1.0                      |
```

Comment:
Naming "GlassFish" as the Dependency is kind of misleading, since we are only using a artifact that happens to be part of the GlassFish project.